### PR TITLE
Add comment for `UserOps`' `sync` method

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: libparsec/crates/client/src/user_ops/mod.rs [hunk 1]
```diff
@@ -82,14 +82,15 @@ impl UserOps {
...
 
-    pub async fn sync(&self) -> Result<(), SyncError> {
-        sync::sync(self).await
-    }
-
     /*
      * Public interface
      */
 
+    // TODO: It has been moved in the public interface, because some of the commands in the CLI didn't sync automatically
```
Comment (FirelightFlagboy) [inline libparsec/crates/client/src/user_ops/mod.rs:89]: What should be done ? `sync` should not be publicly exposed ?

Comment (TimeEngineer) [reply libparsec/crates/client/src/user_ops/mod.rs:89]: I don't know but I have changed something, that's why I mention it

Comment (TimeEngineer) [reply libparsec/crates/client/src/user_ops/mod.rs:89]: It's better to know no ?

Comment (TimeEngineer) [reply libparsec/crates/client/src/user_ops/mod.rs:89]: According to @touilleMan it's something low level that we don't want to expose, but it should be fixed if we want to keep private.

Comment (FirelightFlagboy) [reply libparsec/crates/client/src/user_ops/mod.rs:89]: Then can you update the comment to state that the method is currently expose but it's not something we want ?

Comment (TimeEngineer) [reply libparsec/crates/client/src/user_ops/mod.rs:89]: Sure

Comment (TimeEngineer) [reply libparsec/crates/client/src/user_ops/mod.rs]: Done

**Pull request merged**